README.md

```diff
@@ -63,6 +63,7 @@ After you're finished please remove all the comments and instructions!
 ### Tech Stack <a name="tech-stack"></a>
 
 > React, webpack And babel
```

Comment:
- [ ] As part of the requirements to remove boilerplate files that would not be needed, kindly remove the following files.

![woo](https://user-images.githubusercontent.com/81468675/218699506-7e7b19ac-cde9-4d39-af68-85b7c57dc309.jpg)
![oas](https://user-images.githubusercontent.com/81468675/218699799-62a990c5-36a8-48cd-9708-71abce3b1096.jpg)

Reply:
@julie-ify I appreciate your feedback and made the necessary corrections.